2020-04-29 00:00:01,334 INFO 77049 [egg-logrotator] agent logger reload: got log-reload message
2020-04-29 00:59:04,301 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2143393ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 01:32:58,156 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2010409ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 02:08:00,915 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2055896ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 02:45:50,979 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2250050ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 03:56:22,530 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2238735ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 04:30:27,929 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2022773ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 05:09:30,776 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2322085ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 06:46:47,134 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 1863095ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 07:22:20,008 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 2104076ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 08:00:50,477 ERROR 77049 nodejs.ClusterClientNoResponseError: client no response in 182091ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 77049
hostname: zhangmanludeMBP.lan

2020-04-29 08:01:51,110 INFO 97629 [egg:logger] init all loggers with options: {"dir":"/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/logs/shopping-system-backend","encoding":"utf8","env":"local","level":"INFO","consoleLevel":"INFO","disableConsoleAfterReady":false,"outputJSON":false,"buffer":true,"appLogName":"shopping-system-backend-web.log","coreLogName":"egg-web.log","agentLogName":"egg-agent.log","errorLogName":"common-error.log","coreLogger":{"consoleLevel":"WARN"},"allowDebugAtProd":false,"type":"agent"}
2020-04-29 08:01:51,120 INFO 97629 [egg:core] dump config after load, 4ms
2020-04-29 08:01:51,166 INFO 97629 [egg-watcher] Start watching: ["/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/app","/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/config","/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/mocks","/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/mocks_proxy","/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/app.js"]
2020-04-29 08:01:51,166 INFO 97629 [egg-watcher] Start watching: "/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/app"
2020-04-29 08:01:51,167 INFO 97629 [egg-watcher] Start watching: "/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/config"
2020-04-29 08:01:51,167 INFO 97629 [egg-watcher] Start watching: "/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/mocks"
2020-04-29 08:01:51,167 INFO 97629 [egg-watcher] Start watching: "/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/mocks_proxy"
2020-04-29 08:01:51,167 INFO 97629 [egg-watcher] Start watching: "/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/app.js"
2020-04-29 08:01:51,167 INFO 97629 [egg-watcher:agent] watcher start success
2020-04-29 08:01:51,181 INFO 97629 [egg:core] dump config after ready, 3ms
2020-04-29 21:13:25,757 ERROR 97629 nodejs.ClusterClientNoResponseError: client no response in 41431756ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 97629
hostname: zhangmanludeMacBook-Pro.local

2020-04-29 21:13:29,902 WARN 97629 [agent:development] reload worker because /Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/app/service/goods.js change
2020-04-29 21:13:29,694 INFO 97629 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/zhangmanlu/Desktop/shopping_mall/shopping_egg/app/service/goods.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":33745856,"size":2504,"blocks":8,"atimeMs":1588166009622.2463,"mtimeMs":1588166009629.8726,"ctimeMs":1588166009629.8726,"birthtimeMs":1577355302000,"atime":"2020-04-29T13:13:29.622Z","mtime":"2020-04-29T13:13:29.630Z","ctime":"2020-04-29T13:13:29.630Z","birthtime":"2019-12-26T10:15:02.000Z"},"remove":false,"isDirectory":false,"isFile":true}
